docs: Small changes to API Gateway docs (#12226)
hashicorp/consul: website/content/docs/api-gateway.mdx

   </CodeBlockConfig>

> 1. Create a `values.yaml` file for your Consul API Gateway deployment that contains the following configuration through the Consul Helm chart. See [Helm Chart Configuration - apigateway](https://www.consul.io/docs/k8s/helm#apigateway) for more available options on how to configure your Consul API Gateway deployment via the Helm chart. 

Reviewer: ```suggestion
1. Create a `values.yaml` file for your Consul API Gateway deployment. Copy the content below into the `values.yaml` file. The `values.yaml` will be used by the Consul Helm chart. See [Helm Chart Configuration - apigateway](https://www.consul.io/docs/k8s/helm#apigateway) for more available options on how to configure your Consul API Gateway deployment via the Helm chart. 
```
The first sentence is a run-on sentence, so I broke it up into three sentences that makes it easier for the reader to follow along.
